packages/eas-cli/package.json

```diff
@@ -30,6 +30,7 @@
     "@urql/core": "2.4.3",
     "@urql/exchange-retry": "0.3.2",
     "chalk": "4.1.2",
+    "cli-progress": "3.10.0",
```

Comment:
Any particular reason for picking a different progress bar lib than expo-cli? https://www.npmjs.com/package/progress

Reply:
Yes, `progress` is no longer maintained. 
- https://www.npmjs.com/package/progress Last publish: 3 years ago
- https://www.npmjs.com/package/cli-progress Last publish: 3 months ago